pkgs/by-name/ca/castxml/package.nix

```diff
@@ -66,6 +66,10 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postCheck
   '';
 
+  passthru.tests = testers.testVersion {
```

Comment:
need to add `testers` as an input at the top of the file.

Reply:
Yep, added.